src/DatePicker/index.js

```diff
@@ -68,7 +70,7 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
```

Comment:
You want `aria-controls="datepicker-id-here"`.  Also, `aria-expanded` which is a bool.  Set that to `true` on opening the datepicker